app/javascript/app/router/sections/financial-resources.js

```diff
@@ -0,0 +1,8 @@
+export default [
+  {
+    slug: 'support-needed',
+    label: 'Support needed',
+    path: '/financial-resources/support-needed',
```

Comment:
We are marking the first item as the default path in the others, so support-needed doesn't need to have a sublevel, take a look on the other sections to avoid the issue of the active link.

Reply:
I know, this is just because support needed won't be the default tab. It will be changed when we add support received